webapp/src/components/jira_issue_selector/jira_issue_selector.jsx

```diff
@@ -55,9 +55,14 @@ export default class JiraIssueSelector extends Component {
     };
 
     searchIssues = (text) => {
-        const textEncoded = encodeURIComponent(text.trim().replace(/"/g, '\\"'));
-        const textSearchTerm = (textEncoded.length > 0) ? 'text ~ "' + textEncoded + '*"' : '';
-        const finalQuery = textSearchTerm + ' ' + searchDefaults;
+        const trimmed = text.trim();
+        if (trimmed === '') {
+            return {};
```

Comment:
This function is supposed to return a `Promise<[]>`, so I suggest making the function async, and returning a `[]` here.